dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -105,7 +111,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     }
     void PushNotificationReceivedEventArgs::Handled(bool value)
     {
-        auto lock = m_lock.lock();
+        auto lock = m_lock.lock_exclusive();
```

Comment:
I think this is a shared lock.

Reply:
Is this a setter function we want to be shared?